python/tests/test_framework/test_configs/cb_cont.json

```diff
@@ -0,0 +1,189 @@
+[
+    {
+        "test_name": "cb_two_action",
+        "data_func": {
+            "name": "generate_cb_data",
+            "params": {
+                "num_examples": 100,
+                "num_features": 1,
+                "action_range": [
+                    0,
+                    2
+                ],
+                "reward_function": {
+                    "name": "constant_reward",
+                    "params": {
+                        "reward": [
```

Comment:
let's reflect the fact that it is piecewise constant in the name and move threshold value  either to name or to arguments list